src/view/uniprotkb/components/NumberView.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FC } from 'react';
+import { formatLargeNumber } from '../../../utils/utils';
+
+export enum Unit {
+  DA = 'Da',
+  AA = 'AA',
+}
+
+const NumberView: FC<{ value: number; unit?: Unit }> = ({ value, unit }) => (
+  <span className="number">
```

Comment:
Yes that's the idea...if somehow we decide numbers should be displayed differently. But then it's not really used anywhere else so maybe it should go?...

Reply:
Because it's got a className it made me think i should be able to find the css def for it somewhere. We could either remove it or add a note saying something like "todo: define css class"?